wazimap_ng/general/static/js/variable-filter-widget.js

```diff
@@ -1,15 +1,16 @@
 (function($) {
     jQuery(document).ready(function($) {
         $(function () {
-            $("#variable-permission-filter input[name='variable_type']" ).on('change', ChangeVaribaleValues);
+            $("#variable-permission-filter input[type='radio']" ).on('change', ChangeVaribaleValues);
```

Comment:
Could you fix the spelling mistake - i.e `ChangeVaribaleValues` should be `ChangeVariableValues`